.github/workflows/tests.yml

```
@@ -53,3 +53,19 @@ jobs:
           command: composer install
       - name: Run PHPUnit
         run: vendor/bin/phpunit -v
+  phpstan:
+    name: "PHPStan"
```

The actions are erroring out with the following message:
```
The workflow is not valid. .github/workflows/tests.yml (Line: 57, Col: 5): Required property is missing: runs-on
```

```suggestion
    name: "PHPStan"
    runs-on: ubuntu-latest
```